Tests/RoslynPluginGeneratorTests/AnalyzerPluginGeneratorTests.cs

```diff
@@ -441,18 +680,33 @@ public void CreatePluginManifest_Owners_NotAvailable()
 
         #region Private methods
 
-        private static ProcessedArgs CreateArgs(string packageId, string packageVersion, string language, string sqaleFilePath, bool acceptLicenses, string outputDirectory)
+        private static ProcessedArgs CreateArgs(string packageId, string packageVersion, string language, string sqaleFilePath, bool acceptLicenses, bool recurseDependencies, string outputDirectory)
         {
             ProcessedArgs args = new ProcessedArgs(
                 packageId,
                 new SemanticVersion(packageVersion),
                 language,
                 sqaleFilePath,
                 acceptLicenses,
+                recurseDependencies,
                 outputDirectory);
             return args;
         }
 
+        private void clearDirectory(string path)
```

Comment:
Resolved, no action required - method is no longer used after removing the less-valuable test cases.